frontend/src/App.vue

```diff
@@ -37,6 +39,10 @@ export default {
   },
   created() {
     this.updateVisibility();
+    axios.get("api/csrf-token/").then(response => {
```

Comment:
Use httpClient here, don't create a new instance of axios.

Reply:
Done